arch/Mips/MipsInstPrinter.c

```diff
@@ -121,6 +121,106 @@ static void printRegName(MCInst *MI, SStream *OS, MCRegister Reg)
 	SStream_concat0(OS, Mips_LLVM_getRegisterName(Reg, syntax_opt & CS_OPT_SYNTAX_NOREGNAME));
 }
 
+static void patch_cs_printer(MCInst *MI, SStream *O) {
+	if (MI->csh->syntax & CS_OPT_SYNTAX_NO_DOLLAR) {
+		char *dollar = strchr(O->buffer, '$');
+		if (!dollar) {
+			return;
+		}
+		size_t dollar_len = strlen(dollar + 1);
+		// to include `\0`
+		memmove(dollar, dollar + 1, dollar_len + 1);
+	}
+
+	// replace '# 16 bit inst' to empty.
+	SStream_replc(O, '#', 0);
+	SStream_trimls(O);
+}
+
+static void patch_cs_detail_operand_reg(cs_mips_op *Op, unsigned Reg, unsigned Access) {
```

Comment:
```suggestion
static void patch_cs_detail_operand_reg(cs_mips_op *op, unsigned reg, unsigned access) {
```

Nitpicky, but for Capstone members we used snake case so far. LLVM stuff had camel case.